rmgpy/data/thermo.py

```diff
@@ -1612,15 +1612,27 @@ def __addPolycyclicCorrectionThermoData(self, thermoData, molecule, polyring):
         be applied.
         """
         # look up polycylic tree directly
-        matched_group_thermodata, _, isPartialMatch = self.__addRingCorrectionThermoDataFromTree(None, self.groups['polycyclic'], molecule, polyring)
+        matched_group_thermodata, matched_group, isPartialMatch = self.__addRingCorrectionThermoDataFromTree(None, self.groups['polycyclic'], molecule, polyring)
         
         # if partial match (non-H atoms number same between 
         # polycylic ring in molecule and match group)
         # otherwise, apply heuristic algorithm
         if not isPartialMatch:
-            thermoData = addThermoData(thermoData, matched_group_thermodata, groupAdditivity=True, verbose=True)
-            # By setting verbose=True, we turn on the comments of polycyclic correction to pass the unittest.
-            # Typically this comment is very short and also very helpful to check if the ring correction is calculated correctly.
+            if isBicyclic(polyring):
```

Comment:
Could these if statements be simplified?
```
if isBicyclic(polyring) and matched_group.label in self.groups['polycyclic'].genericNodes:
    estimated_bicyclic_thermodata = self.getBicyclicCorrectionThermoDataFromHeuristic(polyring)
    if not estimated_bicyclic_thermodata:
        estimated_bicyclic_thermodata = matched_group_thermodata
    thermoData = addThermoData(thermoData, estimated_bicyclic_thermodata, groupAdditivity=True)
else:
    thermoData = addThermoData(thermoData, matched_group_thermodata, groupAdditivity=True, verbose=True)
```

Reply:
Yeah, thanks for pointing it out.